src/stream/mod.rs

```diff
@@ -103,7 +103,7 @@ if_std! {
     mod wait;
     mod channel;
     mod split;
-    mod futures_unordered;
+    mod futures_unordered_impl;
```

Comment:
Oh I was thinking that we'd still want a `futures_unordered` module which contained the `FuturesUnordered` type, for example. I think we'll just want to move the definition of the constructor, `futures_unordered`, from the module to this `stream` module

Reply:
No problem, changed!